test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
```

Comment:
Unit tests now rely on `DynamoShapeManager.Preloader` for its geometry preloading activities.